docs/Activation-Status.md

```diff
@@ -4,18 +4,43 @@ PowerAuth Client may need to check for an activation status, so that it can dete
 
 ## Flow of the Activation Status Check
 
-Checking for an activation status is simple. Client needs to prepare a HTTP request with an activation ID. No cryptography is required in this step - in principle, any client can ask for status of any activation ID. Server processes the request and sends back the response with activation status blob. Activation status blob is an encrypted binary blob that encodes activation status. Key `KEY_TRANSPORT` is used to encrypt the activation blob.
+Checking for an activation status is simple. Client needs to prepare a HTTP request with an activation ID and random `STATUS_CHALLENGE`. Server processes the request and sends back the response with activation status blob and random `STATUS_NONCE`. Activation status blob is an encrypted binary blob that encodes activation status. Key `KEY_TRANSPORT` and `STATUS_IV` is used to encrypt the activation blob.
 
 Following sequence diagram shows the activation status check in more detail.
 
 ![Check Activation Status](./resources/images/sequence_activation_status.png)
 
+## Status Blob Encryption
+
+1. Both, client and server calculate `KEY_TRANSPORT_IV` as:
+    ```java
+    SecretKey KEY_TRANSPORT_IV = KDF.derive(KEY_TRANSPORT, 3000)
```

Comment:
Perhaps we should add an explanation why value `3000` is used?

Reply:
The constant is explained in the list of used keys. Just like other similar constants.

Reply:
Ok, I see now.